pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -46,6 +47,8 @@ type attestContext struct {
 const (
 	watchdogInterval  = 15
 	retryTimeInterval = 15
+	//ITokenFile points to a file on tmpfs
+	ITokenFile = "/var/run/eve.integrity_token"
```

Comment:
Better to put this in types/locationconst.go

Reply:
done